platform-api/src/routers/platform.py

```
@@ -0,0 +1,99 @@
+    """Unauthenticated aggregated health check across all external services."""
+    checks = {}
+    
+    async def probe(name: str, coroutine):
+        try:
+            # Enforce 5-second timeout for external probes (AC3)
+            await asyncio.wait_for(coroutine, timeout=5.0)
+            checks[name] = "ok"
+        except Exception as e:
+            checks[name] = f"error: {str(e)}"
+
+    async def check_sanity():
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `check_sanity`

Add return type annotation: `None`

(ANN202)